bagisto/bagisto
# Issue #6991 Fixed

## Issue Reference

#6991 
## Description

UI Issue Fixed
## How To Test This?

Go to Admin -> Catalog -> Products -> Add Product -> Create a new booking product.
Inspect the details in mobile view.
Select the booking type - appointment
Select the Same Slots all days - yes or no.
In both the selections of slots we will be able to see the UI issue.
## Documentation
